best-practices/background_processing.md

```diff
@@ -1,32 +1,11 @@
 # Background Processing at DLSS
 
 There are many tools that provide background processing for Rails/Ruby-based applications. At DLSS,
-we use [Delayed Job](https://github.com/collectiveidea/delayed_job),
-[Resque](https://github.com/resque/resque), [Sneakers](https://github.com/jondot/sneakers) and [Sidekiq](http://sidekiq.org/). We don't have a set
-requirement for which technology to use, but most of us seem to prefer Sidekiq.
-
-
-## Resque
-
-[Resque](https://github.com/resque/resque) is a Ruby application with a companion Sinatra-based
-monitoring web app. It uses [Redis](http://redis.io) as its backend for managing the queues. Each queue is a
-multivalued record (a List data structure) in Redis and Redis handles all the queueing issues like
-concurrency, etc.
-
-Resque workers poll one or more queues to get work. For each job they receive, they fork a process
-to run the job using the self.perform(*args) method. You can run multiple workers per queue if you
-want to have concurrent jobs for a queue. When a worker polls more than 1 queue, it does so in order
-per job which you can leverage to implement priority queues.
-
-Note that per the lead maintainer, Resque is currently
-[in maintenance mode](http://resque.github.io/2016/03/10/resque-1.26.0-released.html).
-
-The DOR Workflow Service framework uses Resque.
-
+we use [Sneakers](https://github.com/jondot/sneakers) and [Sidekiq](http://sidekiq.org/).
```

Comment:
What about racecar / kafka?

Reply:
This is just about getting rid of obsolete stuff. If you want to add something, feel free to create a new PR.